src/main.rs

```diff
@@ -25,29 +32,42 @@ const PING_TIMEOUT: Expiration = OneShot(TimeSpec::new(30, 0));
 const PET_ON: Expiration = OneShot(TimeSpec::new(0, 100_000_000));
 const PET_OFF: Expiration = OneShot(TimeSpec::new(0, 900_000_000));
 
+const GPIO_LABEL: &str = "PET_WDT";
+const GPIO_LINE: u32 = 25;
+const GPIO_CHIP: &str = "gpiochip2";
+const GPIO_CONSUMER: &str = "C3_Watchdog";
+
 // pet every 1s (0.1s high, 0.9s low)
 // wait 120s
 // if port hasn't been pinged in the last 30s, die
 
 // Specifically die to sigterm/sighup/sigint
 // set line low on death
 
+
+fn timestamp_millis() -> u128 {
+    let now = std::time::SystemTime::now();
+    let millis = now.duration_since(UNIX_EPOCH).unwrap().as_millis();
+    return millis;
+}
+
```

Comment:
Since this function is only used under `#[cfg(debug_assertions)]`, it could also have the attribute applied to it, making it clear that it's a debug only function. Then since `UNIX_EPOCH` is only used during debug, its `use` declaration can just be moved inside this function to prevent the unused warning in release mode. `SystemTime` too. The whole thing would look something  like:
```rust
#[cfg(debug_assertions)]
fn timestamp_millis() -> u128 {
    use std::time::{SystemTime, UNIX_EPOCH};
    SystemTime.now()
        .duration_since(UNIX_EPOCH)
        .unwrap()
        .as_millis()
}

```